exercises/bob/.meta/generator/test_template.erb

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+require 'minitest/autorun'
+require_relative '<%= exercise_name %>'
+
+# Common test data version: <%= canonical_data_version %> <%= abbreviated_commit_hash %>
+class <%= exercise_name_camel %>Test < Minitest::Test
```

Comment:
This looks just like the default test template, are there any changes or can it be deleted?